.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# Title
+
+Provide a title that describes
```

Comment:
Do we need this? Since the title is mandatory they won't miss it. Or maybe just say:
"On the Title above, describe what you've ....